css/properties/background-clip.json

```diff
@@ -168,10 +168,15 @@
               "chrome": {
                 "version_added": "3",
                 "partial_implementation": true,
-                "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>)."
+                "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>; see <a href='https://crbug.com/1339290'>bug 1339290</a>)."
               },
               "chrome_android": "mirror",
               "edge": [
+                {
+                  "version_added": "79",
+                  "partial_implementation": true,
+                  "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>; see <a href='https://crbug.com/1339290'>bug 1339290</a>)."
+                },
                 {
                   "version_added": "15"
```

Comment:
```suggestion
                  "version_added": "15",
                  "version_removed": "79"
```

Reply:
Probably good to add a `version_removed` in the below statement as well (I can't add a review comment there because it's too far beyond the PR diff).

Reply:
Changing the test script will affect all future PRs, so this workaround shall suffice for now.